thunder/torch/__init__.py

```diff
@@ -2409,6 +2409,29 @@ def tril_(a: TensorLike, /, diagonal: int = 0, *, fill_value: None | Number = No
     return prims.copy_(tril(a, diagonal, fill_value=fill_value), a)
 
 
+# NOTE triu is the same as tril except that we modify the inequality to return the upper triangular
+# NOTE matrix instead of the lower triangular matrix.Alternatively, triu is tril applied to a transposed input.
```

Comment:
Sorry for the confusion, `triu(x) = tril(x.mT).mT`.

Reply:
I'll remove this from the comment to prevent confusion.